video/out/w32_common.c

```diff
@@ -1964,7 +1964,7 @@ static int gui_thread_control(struct vo_w32_state *w32, int request, void *arg)
     case VOCTRL_GET_WINDOW_ID: {
         if (!w32->window)
             return VO_NOTAVAIL;
-        *(int64_t *)arg = (int64_t)w32->window;
+        *(int64_t *)arg = (int64_t)(intptr_t)w32->window;
```

Comment:
What was the warning here? And does it still warn if you remove `int64` from the double cast? The double cast looks idiotic tbh.

Reply:
> The double cast functionally does nothing and thus looks idiotic tbh.

Double casts are only to make our intent clear to compiler. It is quite common patters, often hid behind macros tho. It was cast of pointer to integer type of different size warning. In this case I can remove `(int64_t)`, because no compiler will complain about up casting int32_t to int64_t with out config.